README.md

```diff
@@ -70,6 +70,20 @@ Get the latest development version. *Not recommended!*
 cordova plugin add https://github.com/NiklasMerz/cordova-plugin-fingerprint-aio.git
 ```
 
+**Android integration (Capacitor and Cordova)**
+
+In you top level (project) ``build.gradle`` file add the maven repository url below.
+This will pull in the required androidx.biometric library adapted for non-androidx apps.
+
+```java
+allprojects {
+    repositories {
+        ...
+        maven { url 'https://jitpack.io' }
```

Comment:
Adding this to the top level is not necessary. You can use src/android/build.gradle to link with jitpack.io. So user will not need to make any additional configuration steps.

Reply:
Hi @exxbrain,
I was wondering if there was a way to do that. Thank you for your suggestion. I will make the requested changes once I'm back in the office on Monday.